packages/components/src/select-control/styles/select-control-styles.ts

```diff
@@ -75,7 +81,7 @@ export const Select = styled.select`
 		${ fontSizeStyles };
 		${ sizeStyles };
 
-		${ rtl( { paddingLeft: 8, paddingRight: 24 } )() }
+		${ rtl( { paddingLeft: 8, paddingRight: 24 } ) }
```

Comment:
Were rtl styles previously not correctly applied correctly? If yes, should this bug fix be included in the changelog?

Reply:
I think they just weren't responsive to other prop changes (but not to `isRtl` changes). I have no idea if this was causing a bug, I'm actually fairly certain that the `rtl` function doesn't really work the way it is intended to work (that is responsively to changes in the `isRtl()` return value... it will only actually update if the component re-renders but nothing in the `rtl` mixin will actually do that.

We might need to do something like create a `withRtl` wrapper to wrap styled components that uses a hook to watch the `RTL` value and re-render appropriately. Right now it'll only update if non-related things cause the component to re-render.

In short, no bug fix here really as the rtl utility doesn't even work correctly to begin with (I think).

Reply:
I'm going to open a PR to explore this issue further, BTW.

Reply:
Thanks for the explanation and for looking further into this. Not a blocker for this PR